Introductory_modules/Introduction_to_Ray.ipynb

```diff
@@ -411,6 +636,8 @@
     "# Homework\n",
     "---\n",
     "If you would like to practice your new skills further with some in-depth examples beyond the embedded coding excercises, take a look at this list of suggested problems:\n",
+    "- [Read About Debugging and Profiling on Ray](https://docs.ray.io/en/latest/ray-core/troubleshooting.html)\n",
+    "    - Dig into how to observe Ray work by visualizing tasks in the Ray timeline, profiling using Python's CProfile, understanding crashes and suboptimal performance, and more in this user guide.\n",
     "- [Distribute a Classical Algorithm with Ray](https://github.com/ray-project/hackathon5-algo)\n",
     "    - In this excercise, go to the GitHub repo linked above for details on choosing a classic algorithm implemented in Python, editing the implementation to parallelize the work with Ray, and compare your results against the sequential implementation.\n",
     "\n",
```

Comment:
Might want add reference to further use of Ray core examples: https://docs.ray.io/en/latest/ray-core/examples/overview.html

Reply:
Done!